xds/src/main/java/io/grpc/xds/LoadReportClient.java

```diff
@@ -386,27 +337,6 @@ private void cleanUp() {
     }
   }
 
-  private final class LoadStatsEntity {
```

Comment:
Eliminated this wrapper. Move the interval recording into each `LoadStatsStore`'s implementation. Since each `LoadStatsStore` represents a "living report", when taking a snapshot for the report, we could "stamp an interval" for the report. Since the report is queried every time for sending an LRS request, it totally makes sense to have the report already stamped with the interval.